collect_app/src/main/res/values/strings.xml

```diff
@@ -314,6 +314,7 @@
 <string name="aggregate_preferences">ODK Aggregate settings</string>
 <string name="other_preferences">Other platform settings</string>
 <string name="google_preferences">Google Sheets settings</string>
+<string name="google_sheets_url_hint"><font size="13">http://docs.google.com/spreadsheets/d/sheet_id</font></string>
```

Comment:
We want to warn people that this field is only for the case where they downloaded a form with no submission URL. 

`Do not use! This is for the unlikely case when a blank form has an empty Google Sheets URL.`